tests/e2e/local/minikube/setup_host.sh

```diff
@@ -14,12 +14,13 @@ case "${OSTYPE}" in
   *) echo "unsupported: ${OSTYPE}" ;;
 esac
 
-# shellcheck disable=SC2153
-if [ ! -z "$VM_DRIVER" ]; then
-  vm_driver="$VM_DRIVER"
-fi
+# VM for Minikube, the default is decided by your os type if not specified
+# e.g.:
+#   hyperkit default to darwin os
+#   kvm2 default to Debian or Ubuntu os
+VM_DRIVER=${VM_DRIVER:-$(vm_driver)}
```

Comment:
Why  can not be:

```
vm_driver=${vm_driver:-$(VM_DRIVER)}
```

?

Reply:
If change like that, i will add `VM_DRIVER=${VM_DRIVER:-""}` , will check later

Reply:
Seems you are right.

I am fine with the change if it's necessary for fixing the shell check errors.